first_script.py

```diff
@@ -31,14 +45,16 @@
 
 # setting it only once to better adapt to local configuration 
 csv_file = os.path.join(data_dir, 'all_connections.csv')
-csv_file_behavior = os.path.join(data_dir, 'behavior2.csv')
+csv_file_supp = os.path.join(data_dir, 'suppl_connections.csv')
+csv_file_behavior = os.path.join(data_dir, 'behavior_8_07022021.csv')
 
 # directory where figures are saved
 write_dir = data_dir
-
+image_dir = '/Users/lebbe/Dropbox/test_nilearn/analyses_septembre2020/EF_plot/' 
```

Comment:
It is not great to point to such path... 
First i'm not sure whether we need it. Second, it would be better if the paths could be made local or relative so that they work on every bod's computer. Besides, writing what files should be where for the script to work would be a nice addition :-)